src/controllers/clinic/clinic.controller.ts

```diff
@@ -1,9 +1,43 @@
-import { fetchClients, getClient } from "../../models/clinic.models";
+import {
+  getClient,
+  fetchSurveys,
+  getTreatment,
+  getProtocol,
+} from "../../models/clients.models";
 import ApiError from "../../utils/ApiError";
 import httpStatus from "http-status";
 import catchAsync from "../../utils/catchAsync";
-import bcrypt from "bcryptjs";
-import jwt from "jsonwebtoken";
-import dotenv from "dotenv";
 
-export default {};
+const getData = catchAsync(async (req: any, res: any) => {
+  const gov_id = req.params.id;
+
+  const client = (await getClient(gov_id))[0]; //gets all client info
+
+  if (!client) {
+    throw new ApiError(httpStatus.BAD_REQUEST, "No client found");
+  }
+  console.log("client", client);
+  const treatment = await getTreatment(client.id);
+  if (!treatment) {
+    const response = {
+      client: client,
+      status: "success",
+    };
+    res.status(httpStatus.OK).send(response);
+  }
+  const protocol = await getProtocol(client.condition);
```

Comment:
the protocol comes from the treatments table, not from a condition.

I really don't care about condition ever, it's just extra data for the clinic.
also what if a condition has multiple protocols ?